### ISSUE TITLE
bilibili error

### ISSUE THREAD

**Issue by DoctorLai** [soimort/you-get#2053]
Please make sure these boxes are checked before submitting your issue – thank you!

- [ ] You can actually watch the video in your browser or mobile application, but not download them with `you-get`.
- [ ] Your `you-get` is up-to-date.
- [ ] I have read <https://github.com/soimort/you-get/wiki/FAQ> and tried to do so.
- [ ] The issue is not yet reported on <https://github.com/soimort/you-get/issues> or  <https://github.com/soimort/you-get/wiki/Known-Bugs>. If so, please add your comments under the existing issue.
- [ ] The issue (or question) is really about `you-get`, not about some other code or project.

Run the command with the `--debug` option, and paste the full output inside the fences:

```
 you-get --debug --itag=18 -t 99 -u --no-caption http://www.bilibili.com/video/av10815497/?tg
[DEBUG] get_content: http://www.bilibili.com/video/av10815497/?tg
[DEBUG] get_content: http://interface.bilibili.com/playurl?&cid=17875194&from=miniplay&player=1&sign=70732b7c26884dacbd79b0f3d4fc5e79
[DEBUG] url_info: http://ws.acgvideo.com/e/84/17875194-1.flv?wsTime=1496249821&platform=pc&wsSecret2=886975c555b85c871b4c6c970509b65f&oi=1318963333&rate=1280
you-get: version 0.4.750, a tiny downloader that scrapes the web.
you-get: ['http://www.bilibili.com/video/av10815497/?tg']
Traceback (most recent call last):
  File "/usr/local/bin/you-get", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.5/dist-packages/you_get/__main__.py", line 92, in main
    main(**kwargs)
  File "/usr/local/lib/python3.5/dist-packages/you_get/common.py", line 1444, in main
    script_main('you-get', any_download, any_download_playlist, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/you_get/common.py", line 1350, in script_main
    download_main(download, download_playlist, args, playlist, stream_id=stream_id, output_dir=output_dir, merge=merge, info_only=info_only, json_output=json_output, caption=caption)
  File "/usr/local/lib/python3.5/dist-packages/you_get/common.py", line 1157, in download_main
    download(url, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/you_get/common.py", line 1437, in any_download
    m.download(url, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/you_get/extractors/bilibili.py", line 187, in bilibili_download
    info_only=info_only)
  File "/usr/local/lib/python3.5/dist-packages/you_get/extractors/bilibili.py", line 101, in bilibili_download_by_cid
    _, type_, temp = url_info(url, headers={'Referer': 'http://www.bilibili.com/'})
  File "/usr/local/lib/python3.5/dist-packages/you_get/common.py", line 420, in url_info
    response = urlopen_with_retry(request.Request(url, headers=headers))
  File "/usr/local/lib/python3.5/dist-packages/you_get/common.py", line 307, in urlopen_with_retry
    return request.urlopen(*args, **kwargs)
  File "/usr/lib/python3.5/urllib/request.py", line 163, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.5/urllib/request.py", line 472, in open
    response = meth(req, response)
  File "/usr/lib/python3.5/urllib/request.py", line 582, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.5/urllib/request.py", line 504, in error
    result = self._call_chain(*args)
  File "/usr/lib/python3.5/urllib/request.py", line 444, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.5/urllib/request.py", line 696, in http_error_302
    return self.parent.open(new, timeout=req.timeout)
  File "/usr/lib/python3.5/urllib/request.py", line 472, in open
    response = meth(req, response)
  File "/usr/lib/python3.5/urllib/request.py", line 582, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.5/urllib/request.py", line 510, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.5/urllib/request.py", line 444, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.5/urllib/request.py", line 590, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 403: Forbidden

```

If there's anything else you would like to say (e.g. in case your issue is not about downloading a specific video; it might as well be a general discussion or proposal for a new feature), fill in the box below; otherwise, you may want to post an emoji or meme instead:

> [WRITE SOMETHING]
> [OR HAVE SOME :icecream:!]

汉语翻译最终日期：2016年02月26日

在提交前，请确保您已经检查了以下内容!

- [ ] 你可以在浏览器或移动端中观看视频，但不能使用`you-get`下载.
- [ ] 您的`you-get`为最新版.
- [ ] 我已经阅读并按 <https://github.com/soimort/you-get/wiki/FAQ> 中的指引进行了操作.
- [ ] 您的问题没有在<https://github.com/soimort/you-get/issues> , <https://github.com/soimort/you-get/wiki/FAQ> 或 <https://github.com/soimort/you-get/wiki/Known-Bugs> 报告，否则请在原有issue下报告.
- [ ] 本问题确实关于`you-get`, 而不是其他项目.

请使用`--debug`运行，并将输出粘贴在下面:

```
[在这里粘贴完整日志]
```

如果您有其他附言，例如问题只在某个视频发生，或者是一般性讨论或者提出新功能，请在下面添加;或者您可以卖个萌:

> [您的内容]
> [舔 :icecream:!]

**Comment by DoctorLai** [comment]
Sometimes, it gets the video link, but the link is 403 forbidden error.

**Comment by rosynirvana** [comment]
ws.acgvideo.com现在检查UA

first reported in #2018
dedicated issue #2021
dup #2022 #2046
fixed in #1698

**Comment by DoctorLai** [comment]
so, is there a workaround? or is it already fixed in the latest version?

**Comment by rosynirvana** [comment]
> fixed in #1698

develop branch is still 0.4.750 release

**Comment by molingyu** [comment]
me too..

**Comment by soimort** [comment]
Closed via #1698.
